rxdogtag-autodispose/build.gradle

```diff
@@ -29,4 +30,15 @@ dependencies {
   testImplementation project(':rxdogtag').sourceSets.test.output
 }
 
+dokka {
+  externalDocumentationLink {
+    url = new URL("http://reactivex.io/RxJava/2.x/javadoc/")
+  }
+
+  outputFormat = 'gfm'
```

Comment:
Can you leave a comment why this format was needed?

Reply:
7a53222